lib/main_sandbox.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:star_movie_app/flavor_config.dart';
+import 'package:star_movie_app/main.dart';
+
+void main() {
+  mainCommon(FlavorConfig(
+    'TraktTv-sandbox',
+    apiBaseUrl: 'https://api-staging.trakt.tv',
```

Comment:
maybe using one file(json, yaml) with all configurations settings is better. Don't know for sure